### PR TITLE
Issues/22

### PR DESCRIPTION
**Pull request by marceldobrica**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -43,4 +43,5 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 ###< symfony/mailer ###
 ARTICLES_ON_PAGE=10
 PROGRAMME_MIN_TIME_MINUTES=15
-PROGRAMME_MAX_TIME_MINUTES=360
\ No newline at end of file
+PROGRAMME_MAX_TIME_MINUTES=360
```
Comment (ovidiuenache) [inline .env]: @marceldobrica this PR holds the fixes from this PR: https://github.com/marceldobrica/training/pull/34

Please create specific and isolated PRs.

**Review by ovidiuenache — CHANGES_REQUESTED** [review]
Please address the feedback above

File: .env (hunk 1 above)
Comment (ovidiuenache) [reply .env]: Pease rebase this one

**Comment by marceldobrica** [comment]
git rebase main issues/22... next git pull ... and push

**Pull request merged**